apps/wallet-dashboard/lib/utils/vesting/vesting.spec.ts

```diff
@@ -125,12 +125,12 @@ describe('vesting overview', () => {
     it('should get correct vesting overview data with timelocked objects', () => {
         const timelockedObjects = MOCKED_SUPPLY_INCREASE_VESTING_TIMELOCKED_OBJECTS;
         const lastPayout = timelockedObjects[timelockedObjects.length - 1];
-        const totalAmount = bigIntRound(
-            (SUPPLY_INCREASE_STAKER_VESTING_DURATION *
-                SUPPLY_INCREASE_VESTING_PAYOUTS_IN_1_YEAR *
+        const totalAmount =
+            (BigInt(SUPPLY_INCREASE_STAKER_VESTING_DURATION) *
```

Comment:
Instead of creating the bigints here, why don't you do it in the definition of the variable instead?

Reply:
because we use as a number in `getSupplyIncrreaseVestingPayoutsCount`, and the duration is a number